iOS使用问题

你好 由于自己编译脚步不过，我使用你提供的变异好的文件，将src拖到iOS项目中，发现#include <cmrc/cmrc.hpp>这个文件找不到。请问怎么解决？

> > 嗯嗯，我主要用的还是这边的代码，参考了[#73](https://github.com/wangfenjin/simple/pull/73)，可以用build-ios.sh编译，也可以在[#73](https://github.com/wangfenjin/simple/pull/73)这个页面找到回复，直接下载“boyan01” 大佬的代码，把Libs这个文件夹拖到项目里，注意用到这个库的地方要把文件改成.mm否则编译不过，代码很少跟simple简介差不多 ` NSString *fts = @" CREATE VIRTUAL TABLE t1 USING fts5(x, tokenize = 'simple')"; BOOL result = [db executeUpdate: fts];`
> > `BOOL result = [db executeUpdate:@"insert into 't1' values(?)" withArgumentsInArray:@[@"周杰伦 Jay Chou:最美的不是下雨天，是曾与你躲过雨的屋檐"]];` `NSString *sql = @"select * from t1 where x match simple_query('zhoujiel')";FMResultSet *result = [db executeQuery:sql];` iOS开发者看到这里基本应该会了，我完成这个功能后也会找时间，写一个文章出来。
> 
> 您好，请问编译build-ios.sh需要修改吗？我这边编译不成功，报错了

CMake Error at src/cmake_install.cmake:74 (file):
  file INSTALL cannot find
  "/Users/dw/Documents/GitHub/simple/build-ios/src/Release-iphoneos/libsimple.a":
  No such file or directory.
Call Stack (most recent call first):
  cmake_install.cmake:62 (include)

@wangfenjin @yugaoxiang
 报错日志：
/Users/daw/Documents/GitHub/simple/build-ios/cppjieba/src/cppjieba/include/cppjieba/QuerySegment.hpp:7:10: fatal error: 
      'limonp/Logging.hpp' file not found
#include "limonp/Logging.hpp"
         ^~~~~~~~~~~~~~~~~~~~
1 error generated.

** BUILD FAILED **


The following build commands failed:
	CompileC /Users/daw/Documents/GitHub/simple/build-ios/src/simple-tokenizer.build/Release-iphoneos/simple.build/Objects-normal/arm64/entry.o /Users/huangxing/Documents/GitHub/simple/src/entry.cc normal arm64 c++ com.apple.compilers.llvm.clang.1_0.compiler (in target 'simple' from project 'simple-tokenizer')
(1 failure)
-- Up-to-date: /Users/daw/Documents/GitHub/simple/./output/bin/dict
-- Up-to-date: /Users/daw/Documents/GitHub/simple/./output/bin/dict/pos_dict
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/pos_dict/prob_trans.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/pos_dict/prob_emit.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/pos_dict/char_state_tab.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/pos_dict/prob_start.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/stop_words.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/hmm_model.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/user.dict.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/idf.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/jieba.dict.utf8
CMake Error at src/cmake_install.cmake:74 (file):
  file INSTALL cannot find
  "/Users/daw/Documents/GitHub/simple/build-ios/src/Release-iphoneos/libsimple.a":
  No such file or directory.
Call Stack (most recent call first):
  cmake_install.cmake:62 (include)

@wangfenjin @yugaoxiang
 报错日志：
/Users/daw/Documents/GitHub/simple/build-ios/cppjieba/src/cppjieba/include/cppjieba/QuerySegment.hpp:7:10: fatal error: 
      'limonp/Logging.hpp' file not found
#include "limonp/Logging.hpp"
         ^~~~~~~~~~~~~~~~~~~~
1 error generated.

** BUILD FAILED **


The following build commands failed:
	CompileC /Users/daw/Documents/GitHub/simple/build-ios/src/simple-tokenizer.build/Release-iphoneos/simple.build/Objects-normal/arm64/entry.o /Users/huangxing/Documents/GitHub/simple/src/entry.cc normal arm64 c++ com.apple.compilers.llvm.clang.1_0.compiler (in target 'simple' from project 'simple-tokenizer')
(1 failure)
-- Up-to-date: /Users/daw/Documents/GitHub/simple/./output/bin/dict
-- Up-to-date: /Users/daw/Documents/GitHub/simple/./output/bin/dict/pos_dict
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/pos_dict/prob_trans.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/pos_dict/prob_emit.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/pos_dict/char_state_tab.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/pos_dict/prob_start.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/stop_words.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/hmm_model.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/user.dict.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/idf.utf8
-- Installing: /Users/daw/Documents/GitHub/simple/./output/bin/dict/jieba.dict.utf8
CMake Error at src/cmake_install.cmake:74 (file):
  file INSTALL cannot find
  "/Users/daw/Documents/GitHub/simple/build-ios/src/Release-iphoneos/libsimple.a":
  No such file or directory.
Call Stack (most recent call first):
  cmake_install.cmake:62 (include)

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -8,6 +8,7 @@ if(SIMPLE_WITH_JIEBA)
       PREFIX ${CMAKE_BINARY_DIR}/cppjieba
       GIT_REPOSITORY https://github.com/yanyiwu/cppjieba.git
       CONFIGURE_COMMAND ""
+      GIT_TAG 194c144d8b5ed1baf3190d07c5226e804454ab47
       BUILD_COMMAND cmake -E echo "Skipping build cppjieba."
       INSTALL_COMMAND cmake -E echo "Skipping install cppjieba."
       LOG_DOWNLOAD ON

